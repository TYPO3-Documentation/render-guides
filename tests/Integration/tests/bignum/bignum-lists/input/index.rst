
==================================
Styled numbered sections (bignums)
==================================

Lorem Ipsum Dolor.

With XXL Big Numbers
====================


..  rst-class:: bignums-xxl

1.  ONE One one bignums-xxl

    Lots of stories here ...

2.  TWO Two two

    Lots of stories here ...


3.  THREE Three three

    *   Well, here we are again, old lovely...
    *   You may now serve the fish.
    *   Fish. Very good, Miss Sophie. Did you enjoy the soup?

With Big Numbers - Tip
======================


..  rst-class:: bignums-tip

1.  ONE One one bignums-tip

    More ...

2.  TWO Two two

    More ...

3.  THREE Three three

    More ...
