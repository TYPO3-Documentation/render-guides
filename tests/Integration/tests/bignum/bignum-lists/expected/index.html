<!-- content start -->
                <section class="section" id="styled-numbered-sections-bignums">
            <h1>Styled numbered sections (bignums)<a class="headerlink" href="#styled-numbered-sections-bignums" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h1>

    <p>Lorem Ipsum Dolor.</p>

            <section class="section" id="with-xxl-big-numbers">
            <h2>With XXL Big Numbers<a class="headerlink" href="#with-xxl-big-numbers" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>


<ol class="bignums-xxl" start="1">
    <li value="1">
    <p>ONE One one bignums-xxl</p>

    <p>Lots of stories here ...</p>
</li>
    <li value="2">
    <p>TWO Two two</p>

    <p>Lots of stories here ...</p>
</li>
    <li value="3">
    <p>THREE Three three</p>


<ul>
    <li>Well, here we are again, old lovely...</li>
    <li>You may now serve the fish.</li>
    <li>Fish. Very good, Miss Sophie. Did you enjoy the soup?</li>
</ul>
</li>
</ol>

    </section>
            <section class="section" id="with-big-numbers-tip">
            <h2>With Big Numbers - Tip<a class="headerlink" href="#with-big-numbers-tip" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h2>


<ol class="bignums-tip" start="1">
    <li value="1">
    <p>ONE One one bignums-tip</p>

    <p>More ...</p>
</li>
    <li value="2">
    <p>TWO Two two</p>

    <p>More ...</p>
</li>
    <li value="3">
    <p>THREE Three three</p>

    <p>More ...</p>
</li>
</ol>

    </section>
    </section>
        <!-- content end -->
